app/views/RoomMembersView/index.js

```diff
@@ -242,8 +245,18 @@ class RoomMembersView extends React.Component {
 		}
 	}
 
+	onCancelPress = () => {
+		this.setState({ filtering: false, membersFiltered: [] });
+		Keyboard.dismiss();
+	}
+
 	renderSearchBar = () => (
-		<SearchBox onChangeText={text => this.onSearchChangeText(text)} testID='room-members-view-search' />
+		<SearchBox
+			onChangeText={text => this.onSearchChangeText(text)}
+			hasCancel={isIOS}
+			onCancelPress={() => this.onCancelPress()}
```

Comment:
same here.

Reply:
We need the onCancelPress function to dismiss  the keyboard, and change to only this.onSearchChangeText will throw an error. Here and in the others.